client/model-builder/pom.xml

```diff
@@ -61,6 +67,45 @@
                     <parameters>true</parameters>
                 </configuration>
             </plugin>
+            <plugin>
+                <artifactId>kotlin-maven-plugin</artifactId>
+                <groupId>org.jetbrains.kotlin</groupId>
+                <version>${version.kotlin.compiler}</version>
+                <executions>
+                    <execution>
+                        <id>compile</id>
+                        <goals>
+                            <goal>compile</goal>
```

Comment:
Don't we only need `test-compile` because you're not using Kotlin in the application?